tests/pipelines/audioldm/test_audioldm.py

```diff
@@ -0,0 +1,816 @@
+# coding=utf-8
```

Comment:
Tests look very nice, but let's maybe not add too many to not slow down the CI too much.

- No nightly test required
- Just 2,3 slow tests (and maybe only one that runs for full 25 steps)
- Let's make sure to not use fp16 in tests as numerical precision is a big problem in testing 
- Let's use a high tolerance (1e-2) as audio tends to be flaky

Reply:
Down to 7 AudioLDM specific fast tests (on top of the 15 standard `PipelineTesterMixin` tests) and 2 slow tests, one of which runs for 25 full inference steps. No nightly tests ✅